modules/Hoot/ui/sidebar/advancedOpts.js

```diff
@@ -161,10 +173,12 @@ export default class AdvancedOpts {
                 .attr( 'id', d => `${d.name}-toggle` )
                 .classed( 'conflate-type-toggle', true );
 
-
             innerInput.merge(innerInputEnter)
                 .property( 'checked', true )
-                .on( 'click', toggleOption );
+                .on('click', function(d) {
+                    let shouldHide = d3.select(this).property('checked');
+                    instance.toggleOption(d, shouldHide);
```

Comment:
The toggleOption function is fired by clicking the label text too. 

So, the func has a weird quality of needing to update the state of the checkbox and so here, when you click a checkbox we don't just rely on vaniilla web browser to decide if the checkbox is checkked or not. We check the checkbox state ourselves and drive it property that way.